nexus/db-queries/src/db/lookup.rs

```diff
@@ -742,6 +750,15 @@ lookup_resource! {
     primary_key_columns = [ { column_name = "id", rust_type = Uuid } ]
 }
 
+lookup_resource! {
+    name = "AddressLot",
+    ancestors = [],
+    children = [], // TODO: Should this include AddressLotBlock?
+    lookup_by_name = true,
+    soft_deletes = true,
+    primary_key_columns = [ { column_name = "id", rust_type = Uuid } ]
+}
+
```

Comment:
We're 100% going to need more lookups than this. I was just starting it off here.